### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by rfarrell961**

### CODE REVIEW

File: frontend/navigation/startWorkoutNav.js [hunk 1]
```diff
@@ -1,21 +0,0 @@
-import React from "react";
-import { NavigationContainer } from "@react-navigation/native";
```
Comment (rubionestor611) [inline frontend/navigation/startWorkoutNav.js:2]: Are you deleting the file with this PR?

File: frontend/navigation/startWorkoutNav.js [hunk 2]
```diff
@@ -1,21 +0,0 @@
-import React from "react";
```
Comment (rubionestor611) [inline frontend/navigation/startWorkoutNav.js:1]: Are you deleting the file?

**Review by rubionestor611 — COMMENTED** [review]
Some comments on here. Other than them perfectly fine

File: frontend/screens/settingsPage.js [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+        setUserMessage(`A confirmation email has been sent to ${email}`);
+    }
+
+    function handleResetPassword()
+    {
+        try
+        {
+            API_Instance
+            .post(`users//forgotpassword/email/send/to`, {
```
[the remark this answers is not in the document]
Comment (rfarrell961) [reply frontend/screens/settingsPage.js:138]: Typo shouldn't matter

File: frontend/navigation/startWorkoutNav.js (hunk 1 above)
Comment (rfarrell961) [reply frontend/navigation/startWorkoutNav.js:2]: Yuh

File: frontend/navigation/startWorkoutNav.js (hunk 2 above)
Comment (rfarrell961) [reply frontend/navigation/startWorkoutNav.js:1]: Yuh

**Review by rubionestor611 — APPROVED** [review]

**Pull request merged**